lib/plug/conn.ex

```diff
@@ -19,6 +19,7 @@ defmodule Plug.Conn do
   * `path_info` - the path split into segments, example: `["hello", "world"]`
   * `script_name` - the initial portion of the URL's path that corresponds to the application
     routing, as segments, example: ["sub","app"]. It can be used to recover the `full_path/1`
+  * `request_uri` - the requested URI path, example: `/trailing/and//double//slashes/`
```

Comment:
I'm +1 on `request_path`, my reasoning being that the RFC allows absolute URIs as well, so calling it `request_uri` makes sense for the RFC, but not for us/cowboy/etc.